src/partials/how_its_made.html

```diff
@@ -2,7 +2,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="../css/how_its_made.css">
```

Comment:
tutaj nie podłączasz. Używasz czcionki z roota, linki są podłączone w index.html, więc najlepiej usuń to (;